io.elementary.Sdk.json

```diff
@@ -160,6 +160,28 @@
                 }
             ]
         },
+        {
+            "name" : "platform-settings",
+            "buildsystem" : "simple",
+            "modules": [
+                {
+                    "name" : "default-gtk-settings",
+                    "buildsystem" : "simple",
+                    "sources" : [
+                        {
+                            "type": "git",
+                            "url": "https://github.com/elementary/default-settings.git"
+                        }
+                    ],
+                    "build-commands": [
+                        "install -Dm644 gtk/settings.ini /usr/etc/gtk-3.0/settings.ini"
```

Comment:
@alatiera are you sure this should be `/usr/etc` not `/usr/local/etc`?

Reply:
I am almost sure that flatpak is mapping `/usr/etc` to `/etc` when mounting the tree, but its not super well documented.